index.js

```diff
@@ -105,7 +142,14 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+  movieByGenreArr = []
+  for (let i = 0; i < movies.length; i ++){
+    if ((movies[i].genre).includes(genre)){
+      movieByGenreArr.push(movies[i])
+    } 
+  }return movieByGenreArr
+}
```

Comment:
Sold work here and we are only missing one test.  Take a look at `toLowerCase()` and think about how we could make this function case insensitive.